### ISSUE TITLE
benchmarking

### ISSUE THREAD

**Issue by microprediction** [unit8co/darts#478]
Hi there. We chatted on LI. This probably belongs in your discourse, sorry, but the invite link appears to be stale. 

With Ian Klasky I've created a sequence-to-sequence interface for DARTS [here](https://github.com/microprediction/timemachines/tree/main/timemachines/skaters/drts) meaning that you can call DARTS functionality like [this](https://github.com/microprediction/timemachines/blob/main/tests/darts/test_darts_random_skater.py)

I don't imagine that would be an add for the darts library (though you are welcome to it) but one advantage is that anything in that format can get sucked into the benchmarking [here](https://microprediction.github.io/timeseries-elo-ratings/html_leaderboards/univariate-k_003.html) automatically, and that might help in some small way determine which libraries or combinations of things might next go in DARTS. (You should see DARTS models in those rankings tomorrow)

Now that the code is sitting [here](https://github.com/microprediction/timemachines/tree/main/timemachines/skaters/drts) it might be obvious to you or DARTS users that some better choices of model/parameter/meta-param should be included.

**Comment by hrzn** [comment]
Hi @microprediction thanks for the information. It would be nice for us if you have any feedback on how we could improve either accuracy wise or in execution speed. Of course, accuracy will depend on the hyper-parameters, and those are typically problem-specific, so default values will only get you so far. But still benchmarks can be interesting. Concerning execution speed, does this measure inference time or both training+inference? In many cases, inference time is the most important and negligible compared to training time. That's especially true for some of the best performing (neural nets) models.

**Comment by microprediction** [comment]
It measures both under the assumption that fitting is required each step. 

I don't know if you have plans to include tbats but it's doing well. 

If you have a good idea about 1,2 or 3 parameter families of hyper-params for any given model I could try to optimize offline using humpday.
